### PR TITLE
Lps x remove portlet request attrs 2 without filter

### PR DESCRIPTION
**Pull request by ChiZelin**

### CODE REVIEW

**Comment by ChiZelin** [comment]
ci:test

**Comment by ChiZelin** [comment]
ci:test

**Comment by ChiZelin** [comment]
ci:retest

File: portal-impl/src/com/liferay/portlet/PortletRequestImpl.java [hunk 1]
```diff
@@ -666,6 +667,30 @@ public void removeAttribute(String name) {
 		_request.removeAttribute(name);
 	}
 
+	public void removePortletRequestAttrs() {
+		HttpServletRequest tempRequest = _request;
```
Comment (dantewang) [inline portal-impl/src/com/liferay/portlet/PortletRequestImpl.java:671]: Recommend: originalRequest

File: portal-impl/src/com/liferay/portlet/PortletContainerImpl.java [hunk 2]
```diff
@@ -393,6 +393,8 @@ private ActionResult _processAction(
 
 			invokerPortlet.processAction(actionRequestImpl, actionResponseImpl);
 
+			actionRequestImpl.removePortletRequestAttrs();
```
Comment (dantewang) [inline portal-impl/src/com/liferay/portlet/PortletContainerImpl.java:396]: Wrap line 394 with try-finally and put the removal in the finally block.

**Pull request closed without merging**